packages/config/config/devices/0x0115/haseman_r4d4.json

```diff
@@ -0,0 +1,726 @@
+// Z-Wave.Me
+// Haseman R4D4
+{
+	"manufacturer": "Z-Wave.Me",
+	"manufacturerId": "0x0115",
+	"label": "Haseman R4D4",
+	"description": "4 x relay and 4 x universal dimmers",
+	"devices": [
+		{
+			"productType": "0xf222",
+			"productId": "0x2111"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Group 1",
+			"maxNodes": 8,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Group 2",
+			"maxNodes": 8
+		},
+		"3": {
+			"label": "Group 3",
+			"maxNodes": 8
+		},
+		"4": {
+			"label": "Group 4",
+			"maxNodes": 8
+		},
+		"5": {
+			"label": "Group 5",
+			"maxNodes": 8
+		},
+		"6": {
+			"label": "Group 6",
+			"maxNodes": 8
+		},
+		"7": {
+			"label": "Group 7",
+			"maxNodes": 8
+		},
+		"8": {
+			"label": "Group 8",
+			"maxNodes": 8
+		},
+		"9": {
+			"label": "Group 9",
+			"maxNodes": 8
+		}
+	},
```

Comment:
These aren't really helpful. Since the device supports Z-Wave+ and Association Group Info CC, I suspect the same info gets auto discovered anyways.
Remove this - unless the auto-discovered labels are way worse, e.g. `GROUP_9`